docker/php-fpm/Dockerfile

```diff
@@ -26,6 +26,7 @@ RUN apk add --no-cache libxml2-dev \
     SYMFONY_ENV=prod composer run-script symfony-scripts && \
     chown -R www-data /var/www/html/var/cache && \
     chown -R www-data /var/www/html/var/logs && \
-    chown -R www-data /var/www/html/var/sessions
+    chown -R www-data /var/www/html/var/sessions && \
+    rm -rf /var/www/html/var/cache/prod/
```

Comment:
Is the Symfony cache warmed back up at a later stage? Otherwise this makes not much sense to me.

Reply:
The configuration is applied when the docker image starts up. The first http hit will then create the correct cache right?

Reply:
I'm not fully sure that also works in `prod` mode, but apparently it does.

Reply:
It does!